.env

```diff
@@ -9,6 +9,4 @@
 # STIB_API = https://proxy.sulliops.co/index.php?http://detobel36.ddns.net/files/osoc/
 
 SERVER_PORT = 8081
-MONGO_HOST=mongodb://test-octotrails123:test-octotrails123@ds231941.mlab.com:31941/mean-octotrails-starter
-MONGO_PORT=18708
```

Comment:
Are you sure those 2 lines need to be removed ?

Reply:
Yes, David said he would fix that. It doesn't work on my machine with the lines in them.

Reply:
Okay, I understand but you shouldn't push it !